po/zh-Hant.po

```diff
@@ -7257,14 +7257,15 @@ msgstr ""
 
 #: src/generics.md:1
 msgid "# Generics"
-msgstr ""
+msgstr "# 泛型"
 
 #: src/generics.md:3
 msgid ""
 "Rust support generics, which lets you abstract an algorithm (such as "
 "sorting)\n"
 "over the types used in the algorithm."
 msgstr ""
+"Rust支援泛型，因此可以抽象化一個演算法（例如：排序）而不需要依賴特定型別。"
```

Comment:
zh-TW style:
* There should be a space between Chinese characters and English letters or numbers. -> `Rust 支援`
* Half-width parentheses `()` should be used instead of full-width parentheses `（）`, even when surrounding Chinese text or surrounded by Chinese text. There shouldn't be spaces between the parentheses and the text within them, but there should be spaces outside of them if adjacent to Chinese characters. -> `算法 (例如：排序) 而`

Reply:
Updated, PTAL. Thanks!